src/utils/processSystem.cpp

```
@@ -10,23 +20,23 @@ ProcessSystem::~ProcessSystem()
 }
-ProcessSystem::ProcessSystem(uint threadNumber)
+ProcessSystem::ProcessSystem(uint _threadNumber)
 {
-    ProcessSystem::threadNumber = threadNumber;
-    ProcessSystem::isRun = false;
+    threadNumber = _threadNumber;
+    isRun = false;
 }
 
-void ProcessSystem::init(vector<path> task, uint maxFailCount = 100)
+void ProcessSystem::init(vector<path> task, uint _maxFailCount)
```

cppcheck

[performance] Function parameter 'task' should be passed by const reference.

(passedByValue)